spec/job/adapters/kubernetes/helper_spec.rb

```diff
@@ -342,8 +344,10 @@
             HOME: '/over/here',
             UID: 1000,
           },
-          memory: '12Gi',
-          cpu: '6',
+          memory_limit: '12Gi',
+          memory_request: '12Gi',
+          cpu_limit: '6',
+          cpu_request: '6',
```

Comment:
Sorry, did that request make it not backward compatible?  I meant just to get rid of `memory` and `cpu` just internally - I think we can still expose & interpret them, at least for now for comparability, but also just for convenience.

Reply:
Those changes are to `Kubernetes::Resources::Container.new` which no longer accepts `cpu` or `memory`.  The actual container YAML can still have those values:

https://github.com/OSC/ood_core/blob/908036a5c4b8cd88afbc587179cff9d7f5e9d468/lib/ood_core/job/adapters/kubernetes/helper.rb#L52-L55

The `Container` class is the internal change which is what the tests had to be updated to account for.  The batch spec still validate old behavior:

https://github.com/OSC/ood_core/blob/908036a5c4b8cd88afbc587179cff9d7f5e9d468/spec/job/adapters/kubernetes/batch_spec.rb#L374-L375